Inc/ReadCombustionTankPressure.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+extern ADC_HandleTypeDef hadc1;
+
+void readCombustionTankPressureTask(void const* arg);
+
+uint16_t getAverageAdc1Reading();
```

Comment:
This doesn't need to be "public", like nothing else should need this function so it doesn't need to be in the header file. I saw it with one your other changes and I missed it then.